src/pages/Proposal/ProposalSummary/Stepper/index.tsx

```diff
@@ -159,8 +159,7 @@ const Stepper: React.FC<IStepperProps> = ({
               </div>
               {step.getTimestamp() && (
                 <Typography variant="tiny" css={styles.dateDefault}>
-                  {t('voteProposalUi.statusCard.dateOnly', { date: step.getTimestamp() })}
-
+                  {t('voteProposalUi.statusCard.dateOnly', { date: step.getTimestamp() })}{' '}
```

Comment:
If we have to hardcode a space I think we should use a nonbreaking space character. `&nbsp;`